qemu/tests/cfg/balloon_stress.cfg

```diff
@@ -1,26 +1,32 @@
 - balloon_stress:
     type = balloon_stress
     virt_test_type = qemu
-    only Windows
     kill_vm = yes
     repeat_times = 500
```

Comment:
Let's change it to 1000.  According to @xiangchunfu, there was a bug found by running several hundreds times. @lijinlijin , is it ok in your side ?